tests/playwright-utils.ts

```diff
@@ -94,6 +99,20 @@ export const test = base.extend<{
 		})
 		await prisma.user.deleteMany({ where: { id: userId } })
 	},
+	i18next: async ({}, use) => {
+		await use(async options => {
+			console.log(options)
+			const lng = options?.lng ?? 'en'
+			const instance = i18next.createInstance()
+			const i18nInit = instance.init({ ns: [lng], ...i18n, ...options })
```

Comment:
I noticed this slows the test a bit https://github.com/epicweb-dev/epic-stack/issues/645 just enough to be significant on my 2019 16" mac with 8% disk space left

I am 90% sure we will have faster tests by initializing i18next in a global setup file https://playwright.dev/docs/test-global-setup-teardown#option-2-configure-globalsetup-and-globalteardown
and then calling createInstance on this i18n instance and passing the options to `i18next.createInstance(options || undefined)`

Indeed the advantage will be that we won't need to await the return of init function anymore in each test

Reply:
actually, I tried global setup and project setup and that's the wrong approach, we should just put the init before the await use, between line 102 and 103

Reply:
I ended up using the following which is performant

```ts
import commonENJson from '#app/translation/en.json' assert { type: 'json' }

export const test = base.extend<{
  ...
  t(key: string): string
}>({
  ...
  t: async ({}, use) => {
    await use((key: string) => {
      // @ts-expect-error -- let's authorize dynamic keys
      return commonENJson?.[key]
    })
  },
  ```

Reply:
I did the change where I put the `i18next.createInstance()` outside of the use function.
But with what you are proposing, are you able to test specific locale ?